playground/terraform/build.gradle.kts

```diff
@@ -363,140 +338,153 @@ tasks.register("deployFrontend") {
     dependsOn(deploy)
 }
 
-/* build, push, deploy Backend app */
-tasks.register("deployBackend") {
+tasks.register<TerraformTask>("setPlaygroundStaticIpAddress") {
     group = "deploy"
-    description = "deploy Backend app"
-    //TODO please add default tag from project_environment property
-    //if !(project.hasProperty("docker-tag")) {
-    //    project.extra.set("docker-tag", project.property("project_environment") as String)
-    //}
-    val config = tasks.getByName("setDockerRegistry")
-    val push = tasks.getByName("pushBack")
-    val deploy = tasks.getByName("terraformApplyAppBack")
```

Comment:
Why don't we need to do this task anymore?

Reply:
It was replaced by `gkebackend` task and not used anywhere